RuleDocumentation/UseApprovedVerbs.md

```diff
@@ -8,6 +8,8 @@ All CMDLets must used approved verbs.
 
 Approved verbs can be found by running the command `Get-Verb`.
 
+Additional documentation on approved verbs can be found in the microsoft docs page [Approved Verbs for PowerShell Commands](https://docs.microsoft.com/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands). If you find the verb you are using is unapproved, try searching the page for the approved equivalent. For example, if you search in the documentation for `Read`, `Open`, or `Search` you will find that the approved verb for those situations is `Get`.
```

Comment:
Ideally this could be broken into [semantic lines](https://sembr.org/).

Reply:
The second sentence is backwards I think; it reads like:

> If you are using an unapproved verb, try searching for its approved equivalent (which you're trying to find)

It might be worded better as:

> Some unapproved verbs are documented on the approved verbs page and point to approved alternatives; try searching for the verb you used to find its approved form. For example, searching for `Search` or `Open` will lead you to `Get`.

Reply:
Good point. I'll update that.